src/installer/installer.go

```diff
@@ -439,6 +443,23 @@ func (i *installer) waitForControlPlane(ctx context.Context) error {
 	return nil
 }
 
+func (i *installer) waitForETCDBootstrap(ctx context.Context) error {
+	i.UpdateHostInstallProgress(models.HostStageWaitingForBootkube, "waiting for ETCD bootstrap to be complete")
+	i.log.Infof("Started waiting for ETCD bootstrap to complete")
+	return utils.WaitForPredicate(waitForeverTimeout, generalWaitInterval, func() bool {
+		// check if ETCD bootstrap has completed every 5 seconds
+		if result, err := i.ops.ExecPrivilegeCommand(nil, "systemctl", "is-active", "progress.service"); result == "inactive" {
```

Comment:
For history, we're waiting for progress.service to complete https://github.com/openshift/installer/blob/191e0707762555796dceb4ecd80f8032d4a664e7/data/data/bootstrap/files/usr/local/bin/report-progress.sh

Which is OCP version independent